litellm/llms/anthropic.py

```diff
@@ -4,20 +4,16 @@
 import time
 from typing import Callable
 from litellm.utils import ModelResponse
+from anthropic import Anthropic, HUMAN_PROMPT, AI_PROMPT
```

Comment:
we had several issues with the anthropic pypi package and are intentionally choosing to POST to their URLs. 

We'd like to avoid using the anthropic package

Reply:
Interesting what are the problems you met ?
It seems like they've recently updated their package making it better.
To answer to @krrishdholakia looking back at the original code and comparing it to the anthropic documentation, it seems that it is supposed to work. However on my side it does not work, it return all the chunk all at once.
I'm going to go back to the original code and try to see how to make it work.
I've tried the following code to see if I didn't have any other unknown problem and it worked:
```
import requests
import json

ANTHROPIC_API_KEY = 'api_key'

headers = {
    'anthropic-version': '2023-06-01',
    'content-type': 'application/json',
    'x-api-key': ANTHROPIC_API_KEY
}

data = {
    'model': 'claude-2',
    'prompt': '\n\nHuman: Tell me a story.\n\nAssistant:',
    'max_tokens_to_sample': 256,
    'stream': True
}

response = requests.post(
    'https://api.anthropic.com/v1/complete',
    headers=headers,
    data=json.dumps(data),
    stream=True
)

# Check if the request was successful
if response.status_code == 200:
    for line in response.iter_lines():
        # Filter out keep-alive new lines
        if line:
            decoded_line = line.decode('utf-8')
            
            # Split each event line by the first colon to get the event type and data
            event_type, event_data = decoded_line.split(":", 1)
            
            if event_type.strip() == "event":
                last_event = event_data.strip()
            
            elif event_type.strip() == "data" and last_event == "completion":
                # Load the data as a JSON object and handle it
                data_json = json.loads(event_data.strip())
                print(data_json)
else:
    print(f"Request failed with status code: {response.status_code}")
```